test/unit/application.spec.js

```diff
@@ -65,20 +65,6 @@ describe('marionette application', function() {
     });
   });
 
-  describe('when an app has been started, and registering another initializer', function() {
```

Comment:
that doesn't solve the problem: who calls the sub-app's initialize method? when? that's the question that needs to be answered, and was previously answered with this behavior. yes it was "bad" design having the sub-app know of the parent app and register itself. i'm still looking for a better design, 3 years later.